schemainspect/pg/sql/domains.sql

```diff
@@ -24,6 +24,5 @@ FROM pg_catalog.pg_type t
 WHERE t.typtype = 'd'
       AND n.nspname <> 'pg_catalog'
       AND n.nspname <> 'information_schema'
-  AND pg_catalog.pg_type_is_visible(t.oid)
```

Comment:
Can you explain this change to me?

Reply:
I was mentioning this the other day, but basically depending what you set your search path to, it will resolve differently. We don't really have any control to the search path that is set by migra when running the visibility checks to see if this domain exists, especially because it is running across multiple schemas. Read more here: https://www.postgresql.org/docs/12/ddl-schemas.html#DDL-SCHEMAS-PATH

At the end of the day, I don't care if it's visible within a search path because we always specify the schema along with the table name. And when comparing db diffs, we shouldn't care about if it was visible to the person querying, just that it exists.